spring-boot-admin-server-ui/src/main/frontend/views/applications/index.vue

```diff
@@ -75,7 +86,12 @@
       statusGroups() {
         const byStatus = groupBy(this.applications, application => application.status);
         const list = transform(byStatus, (result, value, key) => {
-          result.push({status: key, applications: sortBy(value, [application => application.name])})
+          if(this.filter===''){
+            result.push({status: key, applications: sortBy(value, [application => application.name])})
+          }else{
+            const filteredResult=value.filter( val => val.name.includes(this.filter))
```

Comment:
I'd suggest adding conversion to same case letters:
`value.filter( val => val.name.toLowerCase().includes(this.filter.toLowerCase()))`

Reply:
this I added.